### ISSUE TITLE
Machine doesn't output data

### ISSUE THREAD

**Issue by trescube** [openaddresses/machine#610]
The machine process is generating invalid CSV dumps:

- ~~http://results.openaddresses.io/jobs/f0d0415a-886f-4f53-9979-cc092d339f3f~~
- http://results.openaddresses.io/jobs/5c9fb08c-5f00-4c9e-92c0-c15804494fe6

Here's what's output to the .csv file:

```
��>��g!�#indextile_indextilesCREATE UNIQUE INDEX tile_index on tiles (zoom_level, tile_column, tile_row)CcindexnamemetadataCREATE UNIQUE INDEX name on metadata (name)u�ItabletilestilesCREATE TABLE tiles (zoom_level integer, tile_column integer, tile_row integer, tile_data blob)IgtablemetadatametadataCREATE TABLE metadata (name text, value text)
```

**Comment by migurski** [comment]
Definitely a bug, thanks. This looks like the slippy map MBTiles file is being output in place of the output CSV. I’ve been messing with that code path so I probably introduced this problem.

**Comment by migurski** [comment]
I believe this was fixed in version 4.10.1 (https://github.com/openaddresses/machine/pull/607), but this source ran with the slightly earlier version 4.10.0.

**Comment by migurski** [comment]
The re-run with version 4.10.2 worked, so I’m going to close this. Thanks for noticing it, it’s a super subtle bug! I added some tests to make this bug harder to repeat as well.
